test/verify/check-networkmanager-firewall

```diff
@@ -55,6 +55,10 @@ class TestFirewall(NetworkCase):
         m = self.machine
         self.restore_dir("/etc/firewalld", "systemctl try-restart firewalld")
         m.execute("systemctl start firewalld")
+        # Temporary remove the test Dbus port 8765, this workaround can be
+        # removed once the pixel tests are merged and the image updated to not
+        # open port 8765
+        m.execute("firewall-cmd --remove-port=8765/tcp")
```

Comment:
My first guess is that you also need to do this with `--persistent`. Supposedly the page shows the persistent configuration?